larpandoracontent/LArTrackShowerId/SvmPfoCharacterisationAlgorithm.cc

```diff
@@ -5,10 +5,15 @@
  * 
  *  $Log: $
  */
+#ifdef CETLIB_AVAILABLE
```

Comment:
Don't need this any more; won't compile if `CETLIB_AVAILABLE` is actually defined.